packages/ag-charts-community/src/chart/series/polar/donutSeries.ts

```diff
@@ -584,6 +577,13 @@ export class DonutSeries extends PolarSeries<DonutNodeDatum, DonutSeriesProperti
             title.node.translationY = isFinite(dy) ? dy : 0;
         }
 
+        for (const circle of [this.zerosumInnerRing, this.zerosumOuterRing]) {
+            circle.fillOpacity = 0;
+            circle.stroke = this.properties.calloutLabel.color;
```

Comment:
This property isn't available in the constructor, so was using the default fill (which I've removed)